### DISCUSSION TITLE
WalletConnect 2.0 Eth Send transaction unsupported method

### DISCUSSION THREAD

**Discussion by 0x-General** [WalletConnect/walletconnect-monorepo#1566] [Developer Support]
I'm trying to send an `eth_sendTransaction` request in walletconnect 2.0 on the ETH connection. But I'm getting an unsupported method error message 😦

This is how I'm making the request

```
return this.client.request({
  chainId: 'eip155:1',
  topic: this.session.topic,
  request: {
    method: DEFAULT_EIP155_METHODS.ETH_SEND_TRANSACTION,
    params: [{
      ...txParams,
      nonce: encoding.sanitizeHex(encoding.numberToHex(1)),
      data: '0x',
      gasLimit
   }] },
});
```

Any idea why I could be getting this message?

**Comment by chadyj** [comment]
What wallet are you using?

Keep in mind that not all wallets have migrated to v2 yet.

**Reply by 0x-General** [reply]
Hey @chadyj, I'm using Trust Wallet

**Comment by flypaper0** [comment]
hey @0x-General I tested Trust wallet with our [Sample DApp](https://react-app.walletconnect.com/)
`eth_sendTransaction` request successfully received and displayed. I recommend to check [Ethereum RPC Methods docs](https://github.com/WalletConnect/walletconnect-docs/blob/main/docs/advanced/rpc-reference/ethereum-rpc.md) or SDK integration guide. I guess something wrong with params. Could you send a full params map please?

**Comment by llbartekll** [comment]
Does session namespaces contain `eth_sendTransaction` for eip155:1 ?
